src/shims/Directory.Build.props

```diff
@@ -31,4 +31,9 @@
     <GenFacadesOmitType Include="System.Void" />
     <Compile Include="$(MSBuildThisFileDirectory)System.Void.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(MSBuildProjectName)' == 'mscorlib'">
+    <!-- Manually include type-forwards to internal types since GenApi won't
+        generate forwards for them even if they're visible via InternalsVisibleTo. -->
+    <Compile Include="$(MSBuildThisFileDirectory)InternalTypeForwards.cs" />
```

Comment:
Can you rename this to either mscorlib.forwards.cs or something to indicate that it's specific to a single assembly?  Maybe even do something like:
```xml
<!-- include any forwards needed by both generated and manual projects -->
<Compile Condition="Exists('$(MSBuildThisFileDirectory)$(MSBuildProjectName).forwards.cs')"
                 Include="$(MSBuildThisFileDirectory)$(MSBuildProjectName).forwards.cs" />
```

/cc @ViktorHofer in case he has an opinion here.